filelog.go

```diff
@@ -187,12 +189,33 @@ func (w *FileLogWriter) intRotate() error {
 	w.daily_opendate = now.Day()
 
 	// initialize rotation values
-	w.maxlines_curlines = 0
-	w.maxsize_cursize = 0
+	lineCount, _ := lineCounter(w.file)
+	w.maxlines_curlines = lineCount
+	w.maxsize_cursize = lineCount
```

Comment:
Should `cursize` count characters, not lines?

Reply:
Well that's what I get for coding while sick 😓 Yeah it should be the byte count, not the line count